js/ui/handler/scroll_zoom.js

```diff
@@ -97,6 +97,7 @@ ScrollZoom.prototype = {
     },
 
     _zoom: function (delta) {
+        if (delta === 0) return;
```

Comment:
> Touching a trackpad with two fingers was triggering a zoom by 0, which was making icons jump around (switching from nearest neighbour to linear interpolation).

I was thinking that another platform might trigger a zoom by 0.000001 in this scenario